lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Yeah, and again it looks backwards compatible - the setting remains "optional", especially as there's no use of the file validator.